tests/test_single_topology.py

```diff
@@ -673,41 +663,41 @@ def test_setup_intermediate_state_not_unreasonably_slow(arbitrary_transformation
     assert elapsed_time / n_states <= 1.0
 
 
-@pytest.mark.nocuda
-def test_setup_intermediate_bonded_term(arbitrary_transformation):
-    """Tests that the current vectorized implementation _setup_intermediate_bonded_term is consistent with the previous
-    implementation"""
-    st, _ = arbitrary_transformation
-    interpolate_fn = functools.partial(interpolate_harmonic_bond_params, k_min=0.1, lambda_min=0.0, lambda_max=0.7)
-
-    def setup_intermediate_bonded_term_ref(src_bond, dst_bond, lamb, align_fn, interpolate_fn):
-        bond_idxs_and_params = align_fn(
-            src_bond.potential.idxs,
-            src_bond.params,
-            dst_bond.potential.idxs,
-            dst_bond.params,
-        )
+# @pytest.mark.nocuda
+# def test_setup_intermediate_bonded_term(arbitrary_transformation):
```

Comment:
note to self: figure out how/if to re-add these some of these tests - though doesn't seem that useful to begin. It was testing an old ref implementation that is no longer relevant.

Reply:
(yeah doesn't seem to make much sense to keep a version of this - will have to rewrite one from scratch).